### PR TITLE
#94 [ADD] bookmarked posts to Profile Schema

### PR DESCRIPTION
**Pull request by loren589**
Array 'bookmarkedPosts' of post objects added to Profile Schema. Also includes the date the post was posted on.

### CODE REVIEW

File: models/Profile.js [hunk 1]
```diff
@@ -43,6 +43,17 @@ const ProfileSchema = new mongoose.Schema({
             default: Date.now
         }
     }],
+    bookmarkedPosts: [
+    {
+        post: {
```
Comment (gaticosalas) [inline models/Profile.js:48]: rename it to postID

File: models/Profile.js [hunk 2]
```diff
@@ -43,6 +43,17 @@ const ProfileSchema = new mongoose.Schema({
...
+            type: mongoose.Schema.Types.ObjectId,
+            ref: 'post'
+        },
+        date: {
+            type: Date,
+            default: Date.now
+        }
```
Comment (gaticosalas) [inline models/Profile.js:55]: add a picture attribute to the boomarkedPosts object like this:
```
    picture:{
            type: String
    }
```

**Review by gaticosalas — CHANGES_REQUESTED** [review]

**Review by gaticosalas — APPROVED** [review]
good!

**Review by vishybala — APPROVED** [review]

**Comment by MKB-1** [comment]
user story: 4

**Pull request merged**